### DISCUSSION TITLE
Viewing a modal using normal commands

### DISCUSSION THREAD

**Discussion by murtadapy** [Rapptz/discord.py#8962] [Q&A]
Is there a way to lunch a modal using a normal command where it lunches it after sending the command? I couldn't find a way to use `discord.Interaction` in a cog without using a button to trigger the modal

**Comment by Rapptz** [comment]
Nope. Only interactions can send modals.

**Comment by JDJGInc** [comment]
> Is there a way to lunch a modal using a normal command where it lunches it after sending the command? I couldn't find a way to use `discord.Interaction` in a cog without using a button to trigger the modal
> 
> 

If you use buttons or selects, yes
But nothing direct 
Hope this helps

**Reply by murtadapy** [reply]
Thanks for the response! That's more clear now and the documentation was good at explaining that too
